packages/node_modules/@ciscospark/spark-core/test/unit/spec/interceptors/rate-limit.js

```diff
@@ -119,28 +119,28 @@ describe('spark-core', () => {
       describe('#onRequest', () => {
         it('Rejects idbroker request if API is rate limited', () => {
           const future = (new Date().getTime() / 1000) * 2;
-          interceptor.setRateLimitExpiry('https://idbroker.webex.com/horse/v1/myID', future);
-          return interceptor.onRequest({uri: 'https://idbroker.webex.com/horse/v1/myID'}).catch((err) => {
+          interceptor.setRateLimitExpiry(`${process.env.IDBROKER_BASE_URL || 'https://idbroker.webex.com'}/horse/v1/myID`, future);
+          return interceptor.onRequest({uri: `${process.env.IDBROKER_BASE_URL || 'https://idbroker.webex.com'}/horse/v1/myID`}).catch((err) => {
             assert.notCalled(spark.request);
             assert.equal(err.message, 'API rate limited https://idbroker.webex.com/horse/v1/myID');
```

Comment:
Does this need to be changed as well?

Reply:
Looks good. You may want to encapsulate this logic in one `const` since this OR statement is now used in so many places.